setup.py

```diff
@@ -12,14 +12,25 @@
     assert False, "Error: Could not open '%s' due %s\n" % (filepath, error)
 
 
-REQUIRED_PKGS = ["coloredlogs", "sympy", "transformers>=4.15.0", "torch>=1.9", "packaging"]
+REQUIRED_PKGS = [
+    "coloredlogs",
+    "sympy",
+    "transformers>=4.15.0",
+    "torch>=1.9",
+    "packaging",
+    "numpy",
+    "huggingface_hub==0.4.0",
+    "sentencepiece>=0.1.91,!=0.1.92",
+]
 
-TESTS_REQUIRE = ["pytest"]
+TESTS_REQUIRE = ["pytest", "requests", "parameterized", "pytest-xdist"]
 
 QUALITY_REQUIRE = ["black~=22.0", "flake8>=3.8.3", "isort>=5.5.4"]
 
 EXTRAS_REQUIRE = {
-    "onnxruntime": ["onnx", "onnxruntime", "datasets>=1.2.1"],
+    #  pip install -e ".[onnxruntime,dev,intel]"  git+https://github.com/huggingface/transformers.git@main --upgrade
```

Comment:
Delete?

Reply:
Do we have solution for the version incompatibility with intel and ort for building the docs?